README.md

```diff
@@ -2,6 +2,15 @@
 
 FHIR Package Loader is a utility that downloads published FHIR packages from the FHIR package registry.
 
+## FHIR Foundation Project Statement
+
+- Maintainers: This project is maintained by The MITRE Corporation.
+- Issues / Discussion: For FHIR Package Loader issues, such as bug reports, comments, suggestions, questions, and feature requests, visit [FHIR Package Loader GitHub Issues](https://github.com/standardhealth/fhir-package-loader/issues). For discussion of FHIR Shorthand and its associated projects, visit the FHIR Community Chat @ https://chat.fhir.org. The [#shorthand stream](https://chat.fhir.org/#narrow/stream/215610-shorthand) is used for all FHIR Shorthand questions and discussion.
+- License: All contributions to this project will be released under the Apache 2.0 License, and a copy of this license can be found in [LICENSE](LICENSE).
+- Contribution Policy: The FHIR Package Loader Contribution Policy can be found in [CONTRIBUTING.md](CONTRIBUTING.md).
+- Security Information: The FHIR Package Loader Security Information can be found in [SECURITY.md](SECURITY.md).
+- Compliance Information: FHIR Package Loader is designed to work with FHIR packages that are designed for use with FHIR R4, FHIR R4B, or FHIR R5.
```

Comment:
I know that some of this wording probably came from our phone call, but I'd like to avoid the double-use of _designed_. Perhaps:
```suggestion
- Compliance Information: FHIR Package Loader is designed to work with FHIR packages that are based on FHIR R4, FHIR R4B, or FHIR R5.
```